src/Uno.UI.RuntimeTests/Tests/HotReload/Frame/HRApp/MainPage.xaml.cs

```diff
@@ -48,11 +48,10 @@ await RemoteControlClient.Instance.RegisteredProcessors
 
 			var filters = Environment.GetCommandLineArgs().SkipWhile(a => a != "--filters").Skip(1).FirstOrDefault();
 			Console.WriteLine($"Filters = {filters}");
-			UnitTestEngineConfig testConfig = new();
-			if (filters is { Length: > 0 })
+			var testConfig = new UnitTestEngineConfig()
 			{
-				testConfig.Filters = filters?.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries) ?? Array.Empty<string>();
-			}
```

Comment:
The split has been moved?

Reply:
@jeromelaban It's an API change on the runtime test engine package.